src/js/components/invoice/AddItemsPage.jsx

```diff
@@ -364,8 +394,10 @@ class AddItemsPage extends Component {
   validate(values) {
     const errors = {};
     errors.invoiceItems = [];
-
     _.forEach(values?.invoiceItems, (item, key) => {
+      if (_.isNil(_.get(item, 'quantity'))) {
```

Comment:
Basically, I don't see any benefits of using the `_.get` method. Probably it was used, because we didn't have null safe operators before, and this method didn't fail when we tried to get value from a null object (but I could be wrong about that). For now, I would prefer using null safes.
```
if (_.isNil(item?.quantity))
```